crates/driver/src/infra/solver/dto/notification.rs

```diff
@@ -71,9 +75,20 @@ pub enum Kind {
     ScoringFailed(ScoreKind),
     NonBufferableTokensUsed(BTreeSet<H160>),
     SolverAccountInsufficientBalance(U256),
+    DuplicatedSolutionId,
     Settled(Settlement),
 }
 
+#[serde_as]
+#[derive(Debug, Serialize)]
+#[serde(rename_all = "lowercase")]
+pub enum ScoreKind {
+    ZeroScore,
+    ObjectiveValueNonPositive,
+    SuccessProbabilityOutOfRange(f64),
+    ScoreHigherThanObjective { score: U256, objective_value: U256 },
```

Comment:
I think this enum variant probably needs `#[serde(rename_all = "camelCase")]` to adhere to the JSON naming conventions.

Reply:
You are right, fixed.